packages/design-system/src/components/errorFallback/extensionReloadNotification.tsx

```diff
@@ -47,7 +47,7 @@ const ExtensionReloadNotification = ({
               }
             }
           }}
-          text="Refresh panel"
+          text="Refresh Panel"
```

Comment:
Should we translate this as well?
Idea when app is initialised we will instantiate the message in an object according to the locale in the following manner.
```
{
    displayText:'',
    buttonText: ''
}
```
Then we can pass this down to show translated string in the reload notification.